Assets/HoloToolkit-Tests/Input/Shaders/button_unlit_shader.shader

```diff
@@ -52,7 +52,7 @@
 			struct v2f
 			{
 				float4 pos : SV_POSITION;
-				fixed4 uv : TEXCOORD0;
+				fixed3 uv : TEXCOORD0;
```

Comment:
We are only writing to `uv.xy` and `uv.z`. Since uv.w wasn't being written to, this was producing a warning.